### PR TITLE
Enable nuxtjs autodetect

### PR DESCRIPTION
**Pull request by tudorpip**
# Pull Request Template

## Type of change

-   [x] 🍕 New feature
-   [ ] 🐛 Bug Fix
-   [ ] 🔥 Breaking change
-   [ ] 🧑‍💻 Improvement
-   [ ] 📝 Documentation Update

## Description

This PR enables auto-detection for the NuxtJs framework during deployment.

## Checklist

-   [x] My code follows the contributor guidelines of this project;
-   [ ] I have updated the documentation;
-   [ ] I have added tests;
-   [x] New and existing unit tests pass locally with my changes;

### CODE REVIEW

**Review by andreia-oca — APPROVED** [review]
Nice work 🚀 

I have some inline comments.

IMO, we should merge this PR after we are officially supported in Nuxt, so we can directly use `--preset=genezio`.

**Review by costinsin — APPROVED** [review]
I agree with everything Andreia said. Fix those and then this is ready to merge.

File: src/commands/deploy/nuxt.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { GenezioDeployOptions } from "../../models/commandOptions.js";
+import { UserError } from "../../errors.js";
+import { genezioDeploy } from "./genezio.js";
+import { $ } from "execa";
+export async function nuxtJsDeploy(options: GenezioDeployOptions) {
+    await $({ stdio: "inherit" })`npx nuxi build --preset=aws_lambda`.catch(() => {
+        throw new UserError("Failed to build the Nuxt.js project. Check the logs above.");
+    });
+    await genezioDeploy(options);
```
Comment (costinsin) [inline src/commands/deploy/nuxt.ts:9]: Why do you use `genezioDeploy` here?

Comment (tudorpip) [reply src/commands/deploy/nuxt.ts:9]: This was an old PR, no longer relevant.

**Pull request closed without merging**